pkgs/by-name/wi/windows10-icons/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub }:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "windows10-icons";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "B00merang-Artwork";
+    repo = "Windows-10";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-Yz6a7FcgPfzz4w8cKp8oq7/usIBUUZV7qhVmDewmzrI=";
+  };
+
+  installPhase = ''
+    install -Dm644 $(find . -type f -maxdepth 1) -t $out/share/icons/windows10
+    for dir in $(find . -type d -maxdepth 1)
+      do install -d $dir $out/share/icons/windows10/$dir
+    done
+  '';
```

Comment:
Have you tested your derivation?
It looks like its output does not contain even a single icon:

```
❯ tree result
result
└── share
    └── icons
        └── windows10
            ├── 128x128
            ├── 16x16
            ├── 22x22
            ├── 24x24
            ├── 256x256
            ├── 32x32
            ├── 48x48
            ├── 512x512
            ├── 8x8
            ├── index.theme
            ├── README.md
            └── scalable

14 directories, 2 files
```